packages/examples/sdk-server/src/index.ts

```diff
@@ -75,12 +85,12 @@ export const sdkServer = async (): Promise<FastifyInstance> => {
     reply.status(200).send(txRec);
   });
 
-  server.register(baseRoutes, sdkBase);
+  server.register(baseRoutes, { sdkBaseInstance: sdkBase, cacheConfig: configJson.cache });
   server.register(poolRoutes, sdkPool);
   server.register(utilsRoutes, sdkUtils);
   server.register(routerRoutes, sdkRouter);
 
-  server.listen(8080, (err, address) => {
+  server.listen({ port: 8080 }, (err, address) => {
```

Comment:
the port should be config-provided (in accordance with our other server configs)